Source/SwiftLintFramework/Rules/Metrics/EnumCaseAssociatedValuesLengthRule.swift

```diff
@@ -0,0 +1,74 @@
+import SourceKittenFramework
+
+public struct EnumCaseAssociatedValuesLengthRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityLevelsConfiguration(warning: 5, error: 6)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "enum_case_associated_values_count",
+        name: "Enum Case Associated Values Count",
+        description: "Number of associated values should be low",
```

Comment:
maybe elaborate the description a bit more to mention that this is related to enum case declarations?